src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIMiddleware.cs

```diff
@@ -68,8 +68,16 @@ public async Task Invoke(HttpContext httpContext)
 
             var isGet = HttpMethods.IsGet(httpMethod);
 
+            var match = Regex.Match(path, $"^/{Regex.Escape(_options.RoutePrefix)}/?(index.html|index.js)$", RegexOptions.IgnoreCase);
```

Comment:
Refactor to remove the duplication with `index.(html|js)`?

Reply:
✔️